### PR TITLE
Migrate some old docs to readthedocs 

### PR DESCRIPTION
**Pull request by helenlly**
as issue 3585 [https://github.com/intel-analytics/analytics-zoo/issues/3585] request, copy MD files:
 
1.   https://analytics-zoo.github.io/master/#ProgrammingGuide/nnframes/           -->> /docs/readthedocs/source/doc
2.   https://analytics-zoo.github.io/master/#KerasStyleAPIGuide/keras-api-scala/  -->> /docs/readthedocs/source/doc
     and  a linked MD file in keras-api-scala: core.md                                                -->> /docs/readthedocs/source/doc

### CODE REVIEW

**Comment by hkvision** [comment]
Why keras-api-scala? Shouldn't we first migrate python? https://analytics-zoo.github.io/master/#KerasStyleAPIGuide/keras-api-python/

**Comment by hkvision** [comment]
Any preview link of the migration?

**Comment by jason-dai** [comment]
> Why keras-api-scala? Shouldn't we first migrate python? https://analytics-zoo.github.io/master/#KerasStyleAPIGuide/keras-api-python/

I think it will be more useful for BigDL Scala users? Python users can always use TF or PyTorch.

**Comment by hkvision** [comment]
> > Why keras-api-scala? Shouldn't we first migrate python? https://analytics-zoo.github.io/master/#KerasStyleAPIGuide/keras-api-python/
> 
> I think it will be more useful for BigDL Scala users? Python users can always use TF or PyTorch.

Okay, makes sense!

**Comment by helenlly** [comment]
> Any preview link of the migration?

no, it is not added to readthedocs website yet.

File: docs/readthedocs/source/doc/nnframes.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Analytics Zoo provides `NNEstimator` for model training with Spark DataFrame, which
+provides high level API for training a BigDL Model with the Apache Spark
+[Estimator](https://spark.apache.org/docs/2.1.1/ml-pipeline.html#estimators)/
+[Transfomer](https://spark.apache.org/docs/2.1.1/ml-pipeline.html#transformers)
+pattern, thus users can conveniently fit Analytics Zoo into a ML pipeline. The fit result of
+`NNEstimator` is a NNModel, which is a Spark ML Transformer.
+
+please check our
+[NNEstimator API](../APIGuide/PipelineAPI/nnframes.md) for detailed usage.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/nnframes.md:48]: This link is broken

File: docs/readthedocs/source/doc/nnframes.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+**NNClassifier and NNClassifierModel**
+`NNClassifier` and `NNClassifierModel`extends `NNEstimator` and `NNModel` and focus on 
+classification tasks, where both label column and prediction column are of Double type.
+
+**NNImageReader**
+NNImageReader loads image into Spark DataFrame.
+
+please check our
+[ImageProcessing](../APIGuide/PipelineAPI/nnframes.md#NNImageReader) for detailed usage.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/nnframes.md:58]: This link is broken

**Comment by jason-dai** [comment]
Merge `APIGuide/PipelineAPI/nnframes.md` in `nnframes.md`

**Comment by jason-dai** [comment]
And merge `core.md` into `keras-api.md`

File: docs/readthedocs/source/doc/nnframes.md (hunk 2 above)
Comment (helenlly) [reply docs/readthedocs/source/doc/nnframes.md]: > And merge `core.md` into `keras-api.md`

done

**Comment by helenlly** [comment]
> Merge `APIGuide/PipelineAPI/nnframes.md` into `nnframes.md`

done

**Comment by jason-dai** [comment]
Move to `readthedocs/source/doc/UseCase/nnframes.md`, `readthedocs/source/doc/UseCase/keras-api.md`

**Comment by helenlly** [comment]
will modify in a new PR

**Comment by helenlly** [comment]
> Any preview link of the migration?

even without defining them in index.rst,  the pages can be seen via search keywords or  the links:

https://analytics-zoo.readthedocs.io/en/latest/doc/UseCase/nnframes.html
https://analytics-zoo.readthedocs.io/en/latest/doc/UseCase/keras-api.html

FYI. @hkvision

**Pull request merged**